app/src/main/java/org/fossasia/susi/ai/rest/responses/others/RelatedTopics.kt

```diff
@@ -0,0 +1,29 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+/**
+ * <h1>POJO class to parse related topics object in retrofit response from websearch client.</h1>
+
+ * Created by mayank on 09-12-2016.
+ */
+
+class RelatedTopics
+
+(@SerializedName("FirstURL")
+ @Expose
+ var url: String?,
+
+ @SerializedName("Text")
+ @Expose
+ var text: String?,
+
+ @SerializedName("Icon")
+ @Expose
+ var icon: WebIcon?) {
+
+    @SerializedName("Result")
+    @Expose
+    var result: String? = null
+}
```

Comment:
Could you pls tell me why would need to intiliaze the above you sent. We neither have constructor for these in java classes nor in kotlin. But apart of it we have constructor for Web in java as well, thus it may be require to intialize it if needed that why it is there in Kotlin.